joint_limits:
  left_wheel_joint:
      has_velocity_limits    : true
      max_velocity           : 15.0   # m/s
      has_acceleration_limits: true
      max_acceleration       : 5.0   # m/s^2
  right_wheel_joint:
      has_velocity_limits    : true
      max_velocity           : 15.0   # m/s
      has_acceleration_limits: true
      max_acceleration       : 5.0   # m/s^2